shared/src/main/scala/effekt/symbols/symbols.scala

```diff
@@ -233,16 +256,20 @@ package object symbols {
     val ret = Some(Effectful(tpe, Pure))
   }
 
-  sealed trait Effect extends TypeSymbol {
+  /** Effects */
+
+  sealed trait Effect {
```

Comment:
What is the relation between `Effect`, `UserEffect` and `Capabilities`?

Reply:
That is a **very** good question!

`Effect`: something we want to track
`UserEffect`: something that can be handled by `try ... with ...` -- it is only called "UserEffect" because it is userdefinable, wheras `Console` is a "builtin" and can't be handled. We should have a better name ...
`Capability`: a *termsymbol* representing one particular instance of a usereffect. Only usereffects (and state) have corresponding capabilities.